gui/CMakeLists.txt

```diff
@@ -83,16 +80,19 @@ configure_file(include/${SCOPY_MODULE}/${PROJECT_NAME}_config.h.cmakein
 
 target_include_directories(${PROJECT_NAME} INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/include)
 target_include_directories(${PROJECT_NAME} PRIVATE
-    ${CMAKE_CURRENT_SOURCE_DIR}include/${SCOPY_MODULE}/
-    scopy-common
-    scopysigrok-gui)
-target_include_directories(${PROJECT_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include/${SCOPY_MODULE} ${QWT_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS})
+		${CMAKE_CURRENT_SOURCE_DIR}include/${SCOPY_MODULE}/
+		scopy-common
+		scopy-pluginbase
+		scopysigrok-gui)
+target_include_directories(${PROJECT_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include/${SCOPY_MODULE} ${QWT_INCLUDE_DIRS})
```

Comment:
Same error here, please remember to add the ${Boost_INCLUDE_DIRS} variable back in to prevent MacOS errors